homeassistant/components/wevolor/button.py

```diff
@@ -0,0 +1,54 @@
+"""Add 'favorite' button entities for all the shades.
+
+N.b. this is a workaround pending https://github.com/home-assistant/architecture/issues/698
```

Comment:
Let's remove this note from here.

Reply:
is there a better place to put it? I was not sure about the documentation standards; or whether that should be removed entirely?